package NSchemaOrg;

message TNamespace {
    optional string Prefix = 1;
    optional string Namespace = 2;
}

message TTreeNode {
    repeated string Itemtypes = 1;
    repeated string Itemprops = 2;
    optional string Text = 3;
    optional string Href = 4;
    optional string Datetime = 5;
    repeated string Id = 6;
    repeated string Itemref = 7;
    optional uint32 SentBegin = 13;
    optional uint32 SentCount = 14;
    repeated TTreeNode Node = 15;
    repeated TNamespace Namespaces = 16;
}
